students/ryba_paula/lesson_06_dicts_tuples_sets_args_kwargs/list_fantasy_game_inventory.py

```diff
@@ -0,0 +1,22 @@
+from fantasy_game_inventory import displayInventory
+
+
+def addToInventory(inventory, addedItems):
+    for item in addedItems:
+        if item not in inventory:
+            inventory.setdefault(item, 1)
```

Comment:
`setdefault` jest przydatne, gdy nie wiemy czy taki klucz już jest czy nie, więc skoro sprawdziłaś, że klucza nie ma to wystarczy: `inventory[item] = 1` lub użyc sprytnego rozwiązania z zadania frequency_analysis.py, lub zamiast ifów:
inventory.setdefault(item, 0) - ustawiamy 0 jeżeli klucza nie ma
inventory[item] += 1